config/development.yaml

```diff
@@ -82,19 +82,27 @@ dcRedirectionPolicy:
   toDC: ""
 
 archival:
-  status: "enabled"
-  enableReadFromArchival: true
-  defaultBucket: "cadence-development"
-  filestore:
-    storeDirectory: "/tmp/development/blobstore/"
-    defaultBucket: "cadence-development"
-    customBuckets:
-      - "custom-bucket-1"
-      - "custom-bucket-2"
-#  s3store:
-#    region: "us-east-1"
-#    endpoint: "http://127.0.0.1:4572"
-#    s3ForcePathStyle: true
+  history:
+    status: "disabled"
+    enableReadFromArchival: false
+    archiverProvider:
+      filestore:
+        fileMode: "0600"
+        dirMode: "0700"
+      # s3store:
+      #   region: "us-east-1"
+      #   endpoint: "http://127.0.0.1:4572"
+      #   s3ForcePathStyle: true
+  visibility:
+    status: "disabled"
+    enableReadFromArchival: false
+
+domainDefaults:
+  archival:
+    history:
+      defaultStatus: "disabled"
```

Comment:
In a similar way to avoid it should be invalid to provide a URI if status is disabled. Cluster should panic on startup if this config is given.

Reply:
Will be done in the domain config change diff.